src/main/kotlin/interactor/GetCostlierCityWithRentingAnApartmentBetweenCityCenterAndOutsideCityCenterInteractor.kt

```diff
@@ -0,0 +1,49 @@
+package interactor
+
+import model.CityEntity
+import kotlin.math.abs
+import kotlin.math.max
+
+enum class Option {
```

Comment:
1- Change name class to **BedroomOption** .
2- Create package in **interactor** call **utils**.
3- Create file and set name **BedroomOption** and put your class in this file.